src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +30,9 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        
+        this.sequenceNumber = nextSequenceNumber;
```

Comment:
it is good that you ensure the function of the `nextSequenceNumber` follows the semantic of the variable name. If we were to set nextSequenceNumber to 0, then it wouldn't really make sense (unless first person should have zero?).